tests/test_attachment.py

```diff
@@ -0,0 +1,130 @@
+from aiocouch import ConflictError, NotFoundError
+from aiocouch.document import Document
+
+import pytest
+
+# All test coroutines will be treated as marked.
+pytestmark = pytest.mark.asyncio
+
+text = (
+    b"Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed non risus. Suspendisse "
+    b"lectus tortor, dignissim sit amet, adipiscing nec, ultricies sed, dolor. Cras elementum "
+    b"ultrices diam. Maecenas ligula massa, varius a, semper congue, euismod non, mi."
+)
+
+image = (
```

Comment:
I hope that image hasn't hidden satanic text, legal issues or other stuff?

Reply:
Haha, I had a good laugh :grin: 

```shell
python -c '
with open("/tmp/image.webp", "wb") as f:
    f.write(
        b"RIFF\xb0\x00\x00\x00WEBPVP8 \xa4\x00\x00\x00P\x06\x00\x9d\x01*K\x002\x00?\xfd\xfe\xff"
        b"\x7f\xbf\xba\xb6\xb2>H\x03\xf0?\x89b\x00\xcc\x1c\x18\x05i\xc1\xdf\xfde\xe3\xfcr\xcb\xbb"
        b"DX^^\x9f\x9f\xcc/\xd2s]\x9a2j`\xeb\xaa@\x00\xd6\x8c\xfd;\xd5+\x9c\xb8\xb0\x9b\xabM\x07"
        b"\xc4\xd4\x07b\xcc\x1d\x13\x13\xa3\x9dR?\x9aJe\xe6\x1f\xbf\xed\x17\x19\xdc \x17\x92\xd9"
        b"\x8eR\x94\xad\x91+\xf1f@\x81\x9a\xcc{\x89\x84;\xd7\xdf\xffk\xea\xc0\xa8\xb9\xe1\xda\xea"
        b"\xe0\x07\x83\x12&\r\xae)$\x1b\xdc\xe9\xe5\xf0\xb0\xe0a\x0bP\xf1\x04iw\x03a\x86\rM\xe8"
        b"\xe9\xe0i\x08\xb1\xa0\x00\x00\x00")
'
```
```shell
firefox /tmp/image.webp
```